client/a8c-for-agencies/sections/referrals/lib/get-estimated-commission.ts

```diff
@@ -19,60 +19,81 @@ export const getEstimatedCommission = (
 	products: APIProductFamilyProduct[],
 	activityWindow: { start: Date; finish: Date }
 ) => {
-	const totalCommissionInCents = referrals.reduce( ( acc, referral ) => {
-		if ( ! referral?.purchases?.length ) {
-			return acc;
-		}
-		for ( const purchase of referral.purchases ) {
-			// We go over 'active' and 'cancelled' subscriptions
-			if ( ! purchase || purchase.status === 'pending' || purchase.status === 'error' ) {
-				continue;
-			}
-			// Find corresponding product for price
-			const product = products.find( ( product ) => purchase.product_id === product.product_id );
-			if ( ! product ) {
-				continue;
+	const { bdCommissions, legacyCommissionsInCents } = referrals.reduce(
+		( acc, referral ) => {
+			if ( ! referral?.purchases?.length ) {
+				return acc;
 			}
+			for ( const purchase of referral.purchases ) {
+				// We go over 'active' and 'cancelled' subscriptions
+				if ( ! purchase || purchase.status === 'pending' || purchase.status === 'error' ) {
+					continue;
+				}
+				// Find corresponding product for price
+				const product = products.find( ( product ) => purchase.product_id === product.product_id );
+				if ( ! product ) {
+					continue;
+				}
 
-			// Day the license was issued
-			const issuedDate = new Date( purchase.license.issued_at );
-			// Set hours to 0 to compare from start of the day
-			issuedDate.setHours( 0, 0, 0, 0 );
+				if ( purchase.commissions ) {
+					// As of Aug 2025, new client purchases will use BD for purchases
+					// In this case the estimated commission has already been calculated on the backend
+					// and we just need to add it to the total commission
+					acc.bdCommissions += purchase.commissions.estimated_commission_current_quarter ?? 0;
+				} else {
```

Comment:
Most changes in this file are just indention changes and moving a couple lines that are only needed for legacy calculations. We just branch it here depending on it having a BD commission or not. The legacy calculation logic should not have changed at all.